src/core_landice/Registry.xml

```diff
@@ -172,6 +180,46 @@
                         description="If true, then ice from small grounded islands is removed.  Specifically, this finds one- and two-cell masses of ice that are surrounded by open ocean and eliminates them by sending them to the calving flux.  Grounded such ice masses would be islands, floating would be icebergs.  The ice removed is added to the calvingThickness field."
 	                possible_values=".true. or .false."
 		/>
+		<nml_option name="config_finalize_damage_after_advection" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the li_finalize_damage_after_advection function."
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_preserve_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then we preserve the damage before advection (no heal)"
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_calculate_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the calculate_damage function."
```

Comment:
It seems like this should be automatically .true. if you are choosing to use damage-based calving. Is this the fundamental option for making that choice or not? Or, is the idea that you could have 'damagecalving' chosen for the 'calving' option but that you might want to use something like a data (specified) damage field rather than calculating the damage explicitly?

Reply:
Steve was right. The reason I had this option is to consider the case of specified damage data (we don't want to update the damage field during the run).

Reply:
@tongzhangice -- can you clarify then if this is THE singular option for turning the damage calculation on or off? If so, them maybe we need to add a note for that and also clarify somewhere else (e.g., when you specify the calving type?) that this option needs to be toggled on. That is, just selecting "damagecalving" won't actually enable that if this is set to .false. Should the default for this instead be .true. so that we don't have this problem? I.e., only in specialized situations would you want it to be .false.

Reply:
I change the description to "If true, then turn on the calculate_damage function. If false and turn on the damagecalving option, a prescribed damage field should be given and used in the input nc file". I think this way we can be more clear about this option.

Reply:
@tongzhangice , I don't see the changes you describe in the PR for this option either. Did you push the commit with these changes?

Reply:
@matthewhoffman -- I believe discussion around this item can be resolved now (after commit https://github.com/MPAS-Dev/MPAS-Model/pull/747/commits/3505441dfd11230479e53208489bb4e2f5992dee).